config/locales/es.yml

```diff
@@ -0,0 +1,33 @@
+es:
+  eut4g:
+    application_call:
+      fields:
+        name: Nombre
+        email: Email
+        age: Edad
+        gender: Género
+        region: Región/Ciudad
+        country: País
+        applying_as: Estás aplicando como...
+        project_idea: Por favor, describe tu idea de proyecto
+        actions_to_implement: Por favor, describe las acciones que propones implementar
+        project_plan: Por favor, define tu plan de proyecto
+        project_relevance: Según tu punto de vista, ¿cuál es la relevancia del proyecto para una transición verde y justa?
+        group_less_exposed_people: ¿Qué grupo objetivo ("personas menos expuestas") has identificado? ¿Puedes describir brevemente por qué consideras que corren el riesgo de ser excluidos de la transición verde?
+        plan_increase_participation: ¿Cómo planeas aumentar la participgsción de este grupo objetivo? ¿Cómo se beneficiarán de tu proyecto?
```

Comment:
```suggestion
        plan_increase_participation: ¿Cómo planeas aumentar la participación de este grupo objetivo? ¿Cómo se beneficiarán de tu proyecto?
```

Reply:
Nice! I have changed the form with these recommendations and I have updated the locales too